configBase.js

```diff
@@ -14,15 +14,18 @@ const API_USERNAME = process.env.API_USERNAME
 const SSL = process.env.SSL
 const MIME_TYPE = process.env.MIME_TYPE
 const CONFIG_FILE = process.env.CONFIG_FILE
+const ADDITIONAL_HEADERS = process.env.ADDITIONAL_HEADERS ?? ''
 
 let data
 let dataHeaders = {}
 
 switch (MIME_TYPE) {
   case 'multipart/form-data':
     data = new FormData()
-    data.append('form', fs.createReadStream(path.resolve(__dirname, CONFIG_FILE)))
+    data.append('file', fs.createReadStream(path.resolve(__dirname, CONFIG_FILE)))
+
     dataHeaders = data.getHeaders()
+    ADDITIONAL_HEADERS ? Object.assign(dataHeaders, JSON.parse(ADDITIONAL_HEADERS)) : ""
```

Comment:
```suggestion
    ADDITIONAL_HEADERS && Object.assign(dataHeaders, JSON.parse(ADDITIONAL_HEADERS))
```
Use ternaries when setting a variable, not as a general `if` since that empty string at the end is meaningless.

Reply:
Make sure this works, there can be weird edge cases with truthiness

Reply:
Additionally I'd recommend using an if block for this instead of the &&  or || shorthands, as @tumbledwyer mentioned, javascript's strange truthiness here could lead to a bug.

Also, fun fact, you should be able to use the spread operator here instead of Object.assign, the spread operator (...) is a more recent syntax, its a part of the ES6 standard, worth looking into. 
eg. 
```
if(ADDITIONAL_HEADERS) {
  dataHeaders={...dataHeaders, ...JSON.parse(ADDITIONAL_HEADERS)}
}
```